lib/tasks/pipeline_monitor.rake

```diff
@@ -38,7 +38,8 @@ class CheckPipelineRuns
     # Force refresh well before autoscaling.EXPIRATION_PERIOD_MINUTES.
     # prod does it more often because it needs to pick up updates from
     # all other environments and adjust the autoscaling groups.
-    Rails.env == "prod" ? 60 : 600
+    cloud_env = ["prod", "staging"].include?(Rails.env)
+    Rails.env == cloud_env ? 60 : 600
```

Comment:
staging is now an independent clound env and we should make a scaling decision for it every 60 seconds